style.css

```diff
@@ -3,490 +3,355 @@
+    line-height: 24px;
+    color: #6b778c;
+  }
+
+  .circle {
+    list-style-type: none;
+    display: flex;
+    gap: 10px;
+  }
+
+  .circle li {
+    padding-left: 5px;
+    padding-top: 10px;
+  }
+
+  .underline {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    padding-top: 26px;
+  }
+
+  .works-image {
+    height: 100%;
+    width: 100%;
+  }
+
+  .lang {
+    list-style-type: none;
+    display: flex;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+    padding-top: 20px;
+    gap: 20px;
+  }
+
+  .lang li {
+    background-color: #ebebff;
+    border-radius: 8px;
+    padding: 7px 8px 7px 8px;
+  }
+
+  button {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    box-sizing: border-box;
+    padding: 12px 18px;
+    background: #fff;
+    border: 1px solid #435c9d;
+    border-radius: 14px;
+    margin-top: 30px;
+    transition: 0.036s;
+  }
+
+  button:hover {
+    background-color: #6070ff;
+    color: #fff;
+  }
+
+  .color {
+    background-color: #396df2;
+    color: #fff;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: auto;
+    margin-bottom: 36px;
+  }
+
+  .about-card {
+    display: flex;
+  }
+
+  .about {
+    border-radius: 0 60px 0 0;
+    padding: 30px;
+  }
+
+  .ME {
+    margin-left: 70px;
+  }
+
+  .languages {
+    margin: 150px 0 70px 30px;
+  }
+
+  h5 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .fitems {
+    display: flex;
+    gap: 3%;
+  }
+
+  .fitems li {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #253858;
+  }
+
+  .item-1 li {
+    background: #f7f7f9;
+    border-radius: 8px;
+    width: 122px;
+    height: 90px;
+    margin-top: 30px;
+    margin-bottom: 30px;
+    padding: 7px 0 10px 9px;
+  }
+
+  .photo li {
+    gap: 90%;
+  }
+
+  .photo-1 li {
+    gap: 90%;
+    padding-top: 40px;
+    padding-bottom: 10px;
+  }
+
+  .photo-2 li {
+    gap: 100%;
+    padding-top: 40px;
+    padding-bottom: 10px;
+  }
+
+  .languages ul li {
+    list-style-type: none;
+    display: flex;
+  }
+
+  .contact-me {
+    display: flex;
+    background-image: url("imgs/contact\ form\ background\ shapes.png");
+    background-color: #6070ff;
+    color: #fff;
+    align-items: center;
+    flex-direction: column;
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-top: 103px;
+    padding-bottom: 45px;
+    border-radius: 60px 0 0 0;
+    text-align: center;
+  }
```

Comment:
- [ ] Kindly correct the size of the inputs to look similar to the Figma template. 👇🏽 

| **Figma template** | **Your desing** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/193380018-80fe32b4-eaa4-4129-95a6-6dc0e5c5234a.png) | ![image](https://user-images.githubusercontent.com/58642949/193380030-c7b75af2-a2bf-4404-92f8-e169c89ffb57.png) |